docs/source/_json/lock.resp

```diff
@@ -0,0 +1,12 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "creator": "admin", 
+  "locked": true, 
+  "name": "plone.locking.stealable", 
+  "stealable": true, 
+  "time": 1477076400.0, 
```

Comment:
@buchi yeah. I got that you just mapped over the attributes. This is what we did everywhere so far. On one hand an API should not expose internal data structures, on the other hand re-inventing new names might confuse devs even more. I am ok with leaving this as it is. I just wanted to bring it up.